src/singletons/Theme.cpp

```diff
@@ -66,216 +48,152 @@ Theme::Theme()
+    this->messages.textColors.regular = isLight ? "#000" : "#fff";
+    this->messages.textColors.link = QColor(66, 134, 244);
     this->messages.textColors.system = QColor(140, 127, 127);
     this->messages.textColors.chatPlaceholder =
-        isLight_ ? QColor(175, 159, 159) : QColor(93, 85, 85);
+        isLight ? QColor(175, 159, 159) : QColor(93, 85, 85);
 
-    this->messages.backgrounds.regular = getColor(0, sat, 1);
-    this->messages.backgrounds.alternate = getColor(0, sat, 0.96);
+    this->messages.backgrounds.regular = getGray(1);
+    this->messages.backgrounds.alternate = getGray(0.96);
 
-    // this->messages.backgrounds.resub
-    // this->messages.backgrounds.whisper
-    this->messages.disabled = getColor(0, sat, 1, 0.6);
-    // this->messages.seperator =
-    // this->messages.seperatorInner =
+    this->messages.disabled = getGray(1, 0.6);
 
-    int complementaryGray = this->isLightTheme() ? 20 : 230;
     this->messages.highlightAnimationStart =
-        QColor(complementaryGray, complementaryGray, complementaryGray, 110);
+        QColor::fromHslF(0, 0, isLight ? 0.9 : 0.08, 0.43);
     this->messages.highlightAnimationEnd =
-        QColor(complementaryGray, complementaryGray, complementaryGray, 0);
+        QColor::fromHslF(0, 0, isLight ? 0.9 : 0.08, 0);
 
     // Scrollbar
     this->scrollbars.background = QColor(0, 0, 0, 0);
-    //    this->scrollbars.background = splits.background;
-    //    this->scrollbars.background.setAlphaF(qreal(0.2));
-    this->scrollbars.thumb = getColor(0, sat, 0.70);
-    this->scrollbars.thumbSelected = getColor(0, sat, 0.65);
-
-    // tooltip
-    this->tooltip.background = QColor(0, 0, 0);
-    this->tooltip.text = QColor(255, 255, 255);
+    this->scrollbars.thumb = getGray(0.70);
+    this->scrollbars.thumbSelected = getGray(0.65);
 
     // Selection
-    this->messages.selection =
-        isLightTheme() ? QColor(0, 0, 0, 64) : QColor(255, 255, 255, 64);
-
-    if (this->isLightTheme())
-    {
-        this->splits.dropTargetRect = QColor(255, 255, 255, 0x00);
-        this->splits.dropTargetRectBorder = QColor(0, 148, 255, 0x00);
-
-        this->splits.resizeHandle = QColor(0, 148, 255, 0xff);
-        this->splits.resizeHandleBackground = QColor(0, 148, 255, 0x50);
-    }
-    else
-    {
-        this->splits.dropTargetRect = QColor(0, 148, 255, 0x00);
-        this->splits.dropTargetRectBorder = QColor(0, 148, 255, 0x00);
-
-        this->splits.resizeHandle = QColor(0, 148, 255, 0x70);
-        this->splits.resizeHandleBackground = QColor(0, 148, 255, 0x20);
-    }
-
-    this->splits.header.background = getColor(0, sat, flat ? 1 : 0.9);
-    this->splits.header.border = getColor(0, sat, flat ? 1 : 0.85);
+    this->messages.selection = QColor::fromHsl(0, 0, isLight ? 0 : 255, 54);
+
+    // Splits
+    this->splits.dropTargetRect = QColor(0, 148, 255, 0);
+    this->splits.dropTargetRectBorder = QColor(0, 148, 255, 0);
```

Comment:
Let's keep the light/dark theme colors of `dropTargetRect` even if their alpha is 0